package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@titanium-sdk/ti.map",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
I assume you did this manually, because the package-lock.json wasn't also updated. You'll need to update that files as well.

Next time, use `npm --no-git-tag-version version patch` and it'll update both.